### PR TITLE
add simpeg

### PR DESCRIPTION
**Pull request by sgkang**
first trial adding simpeg to conda-forge

### CODE REVIEW

File: recipes/simpeg/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+  # requires:
+    # Put any additional test requirements here.  For example
+    # - nose
+
+about:
+  home: http://simpeg.xyz/
+  license: MIT
+  summary: 'SimPEG: Simulation and Parameter Estimation in Geophysics'
+  license_family: MIT
```
Comment (synapticarbors) [inline recipes/simpeg/meta.yaml]: please add

```
dev_url: https://github.com/simpeg/simpeg
doc_url: http://docs.simpeg.xyz/index.html
```

File: recipes/simpeg/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,95 @@
+package:
```
Comment (synapticarbors) [inline recipes/simpeg/meta.yaml]: You might consider using jinja template variables as in:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L1-L19

File: recipes/simpeg/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+  run:
+    - python
+    - numpy >=1.7
+    - scipy >=0.13
+    - cython
+    - pymatsolver >=0.1.1
+    - ipython
+    - matplotlib
+    - properties[math] >=0.3.1b2
```
Comment (synapticarbors) [inline recipes/simpeg/meta.yaml]: You should drop the `[math]` part of `properties`. The conda-forge recipe is built with the math extras requirements already. 

You're going to have to add recipes for `discretize` and `pymatsolver` as I don't think they are available on conda-forge or in the `defaults` channel.

File: recipes/simpeg/bld.bat [hunk 4]
```diff
@@ -0,0 +1,8 @@
+"%PYTHON%" setup.py install
```
[the remark this answers is not in the document]
Comment (sgkang) [reply recipes/simpeg/bld.bat]: thanks @synapticarbors I deleted them.

File: recipes/simpeg/meta.yaml (hunk 2 above)
Comment (sgkang) [reply recipes/simpeg/meta.yaml]: done

File: recipes/simpeg/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,95 @@
+package:
+  name: simpeg
+  version: "0.6.2b1"
+
+source:
+  fn: SimPEG-0.6.2b1.tar.gz
+  url: https://pypi.python.org/packages/1b/dd/1ce3fb26388d5a5fec6e4604679721fe3de0cdc0e5242438f64caaf21c7d/SimPEG-0.6.2b1.tar.gz
```
[the remark this answers is not in the document]
Comment (sgkang) [reply recipes/simpeg/meta.yaml]: done

File: recipes/simpeg/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+  md5: d402a75bc696a5ddd3ffb424b30eda8a
```
[the remark this answers is not in the document]
Comment (sgkang) [reply recipes/simpeg/meta.yaml]: done

File: recipes/simpeg/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+#  patches:
```
[the remark this answers is not in the document]
Comment (sgkang) [reply recipes/simpeg/meta.yaml]: done

File: recipes/simpeg/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+   # List any patch files here
+   # - fix.patch
+
+build:
+  number: 0
```
[the remark this answers is not in the document]
Comment (sgkang) [reply recipes/simpeg/meta.yaml]: done

File: recipes/simpeg/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+
+requirements:
+  build:
```
[the remark this answers is not in the document]
Comment (sgkang) [reply recipes/simpeg/meta.yaml]: that is true.

File: recipes/simpeg/meta.yaml (hunk 3 above)
Comment (sgkang) [reply recipes/simpeg/meta.yaml]: right. we need to do that.

File: recipes/simpeg/meta.yaml [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+  # You can also put a file called run_test.py in the recipe that will be run
+  # at test time.
+
+  # requires:
+    # Put any additional test requirements here.  For example
+    # - nose
+
+about:
+  home: http://simpeg.xyz/
+  license: MIT
```
[the remark this answers is not in the document]
Comment (sgkang) [reply recipes/simpeg/meta.yaml]: done

File: recipes/simpeg/meta.yaml (hunk 1 above)
Comment (sgkang) [reply recipes/simpeg/meta.yaml]: done

File: recipes/simpeg/meta.yaml [hunk 11]
```diff
@@ -0,0 +1,95 @@
...
+  summary: 'SimPEG: Simulation and Parameter Estimation in Geophysics'
+  license_family: MIT
+
+extra:
+  recipe-maintainers:
+    - rowanc1
```
[the remark this answers is not in the document]
Comment (sgkang) [reply recipes/simpeg/meta.yaml]: temporarily delete them, still guess they are willing to.

File: recipes/simpeg/meta.yaml [hunk 12]
```diff
@@ -0,0 +1,79 @@
...
+  license: MIT
+  summary: 'SimPEG: Simulation and Parameter Estimation in Geophysics'
+  license_family: MIT
+  license_file: LICENSE.txt
+  dev_url: https://github.com/simpeg/simpeg
+  doc_url: http://docs.simpeg.xyz/index.html
+
+extra:
+  recipe-maintainers:
+    - sgkang
```
Comment (lheagy) [inline recipes/simpeg/meta.yaml]: Thanks @sgkang!

**Comment by sgkang** [comment]
will be handled in different repo

**Pull request closed without merging**